### ISSUE TITLE
How to control individual key or group of keys styles? Is this possible?

### ISSUE THREAD

**Issue by otaviocv** [russellgoldenberg/svelte-keyboard#6]
Hi there! I was playing around with this component to apply individual styles for individual keys. I know it is possible to apply styles with:
```
:global(.svelte-keyboard .key--q) {
        background-color: rgb(255, 0, 234);
    }
```

But in the case I want to build a wordle-like game, how would I control dynamically styles in group of keys? Like displaying `A`, `B` and `C` as "correct", "green" letters. I was thinking to apply a `.correct` or `.missed` classes, but as far as I could read the code there is no interface for the user to control these things.

Is this possible?

I would like already to thank you for your time and attention.

**Comment by russellgoldenberg** [comment]
good idea! just implemented and added docs at the bottom of the the [custom style section](https://github.com/russellgoldenberg/svelte-keyboard#custom-styles) and a [demo](https://russellgoldenberg.github.io/svelte-keyboard/) at the bottom as well. Let me know what you think.

**Comment by russellgoldenberg** [comment]
By the way it is now published in v0.4.0

**Comment by otaviocv** [comment]
Thank you a lot! That was exactly the implementation was thinking. Thank you again! 🎊
